src/openrct2/ride/water/SplashBoats.cpp

```diff
@@ -810,14 +826,19 @@ static void PaintSplashBoatsTrackLeftQuarterTurn5Tiles(
 
     if (trackSequence != 1 && trackSequence != 4)
     {
-        static constexpr uint8_t supportTypes[][7] = {
-            { 0, 0xFF, 5, 3, 0xFF, 5, 1 },
-            { 1, 0xFF, 2, 4, 0xFF, 2, 0 },
-            { 0, 0xFF, 3, 5, 0xFF, 3, 1 },
-            { 1, 0xFF, 4, 2, 0xFF, 4, 0 },
+        static constexpr std::optional<WoodenSupportSubType> supportTypes[NumOrthogonalDirections][7] = {
```

Comment:
This is a bit of an odd one as it never asks if it holds a value or not, it's probably better to have a `None/Invalid` value for the gaps.